query/clusters/src/cluster.rs

```diff
@@ -39,6 +39,7 @@ impl Cluster {
         let mut node = Node {
             name: n.name.clone(),
             cpus: n.cpus,
+            priority: n.priority,
```

Comment:
Would be better to add some comments about priority, such as what's the rule and how we set the value

Reply:
Is the comment good enough?

Reply:
LGTM.
255 granularity is too fine? Maybe [0-10] is enough.

Reply:
Changed.